<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debit</class>
 <widget class="QWidget" name="debit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>debit</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная стака</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма на вкладе в конце срока</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSum">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePlus">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>125</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePersentRate">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushStart">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QPushButton:open { /* when the button has its menu open */
        /*background-color: qlineargradient(x1: 1, y0: 1, x2: 1, y2: 1,
                                           stop: 1 #535353 stop: 1 #535353);*/
      background-repeat: repeat-y;
      background-position: left;

     }

  QPushButton{
     	border-color: darkblue;
		border-style: outset; /*groove*/
		color: palette(dark);
}

     QPushButton::menu-indicator {
         image: url(menu_indicator.png);
         subcontrol-origin: padding;
         subcontrol-position: bottom right;
     }



QPushButton:hover { color: green } /*при наведении*/</string>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineFinalTax">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>391</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lineFinalSum">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lineFinalPercent">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkCapital">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QPushButton:open { /* when the button has its menu open */
        /*background-color: qlineargradient(x1: 1, y0: 1, x2: 1, y2: 1,
                                           stop: 1 #535353 stop: 1 #535353);*/
      background-repeat: repeat-y;
      background-position: left;

     }

  QPushButton{
     	border-color: darkblue;
		border-style: outset; /*groove*/
		color: palette(dark);
}

     QPushButton::menu-indicator {
         image: url(menu_indicator.png);
         subcontrol-origin: padding;
         subcontrol-position: bottom right;
     }



QPushButton:hover { color: orange } /*при наведении*/</string>
   </property>
   <property name="text">
    <string>Стереть </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMinus">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>164</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPlus">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Без пополнения</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пополнение раз в год </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пополнение раз в полугодие</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пополнение раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пополнение раз в месяц</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxMinus">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Без снятия</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Снятие раз в год </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Снятие раз в полугодие</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Снятие раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Снятие раз в месяц</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxPeriod">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>60</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1 месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 месяца</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 лет</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
